pkg/server/server.go

```diff
@@ -73,11 +72,6 @@ func New(opt Opt) (*Server, error) {
 		fmt.Fprintf(os.Stderr, "Unable to connect to database: %v\n", err)
 		os.Exit(1)
 	}
-	err = util.ApplySchema(conn)
```

Comment:
Why skip apply schema?

Reply:
Because we use migration tools for now. This is a simple function to call the schema.sql, which may have error if the table already exists

Reply:
I got it.